addon/components/frost-textarea.js

```diff
@@ -69,10 +66,6 @@ export default Component.extend(FrostEventsProxy, PropTypeMixin, {
 
   init () {
```

Comment:
Since we are no longer using `init() {}` to do some logic during initialization, the following can be removed:

```
init () {
  this._super(...arguments)
},
```